praetorian_cli/sdk/chariot.py

```
@@ -213,12 +219,21 @@ def download(self, name: str, global_=False) -> bytes:
         if global_:
             params |= GLOBAL_FLAG
 
-        resp = self._make_request('GET', self.url('/file'), params=params, allow_redirects=True)
+        resp = self.chariot_request('GET', self.url('/file'), params=params)
+        process_failure(resp)
+
+        data = resp.json()
+        url = data.get("url", None)
+        if not url:
+            message = f'Download request failed: response missing URL' + (f'\nBody: {resp.text}' if resp.text else '(empty)')
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)